src/gui/QvisExpressionsWindow.C

```diff
@@ -817,6 +818,11 @@ QvisExpressionsWindow::SetStandardEditorReadOnly(bool read_only)
 // Creation:   2010 Thu Feb 11 10:47:51 PST 2010
 //
 // Modifications:
+//    Cyrus Harrison, Tue May 20 09:11:37 PDT 2025
+//    Disabled rich text paste support in expression editor text box.
+//    Many folks are now copying and pasting from editors that embed
```

Comment:
Is this comment in the wrong place?

Reply:
Confused by the github diff.
line 821 is for the `CreatePythonFilterEditor` function, not `SetStandardEditorReadOnly`

Reply:
But I don't see any changes in  `CreatePythonFilterEditor`.